### PR TITLE
#30 Descriptions don't include linebreaks

### LINKED ISSUE #30: descriptions don't respect paragraphs
Screenshot 2024-09-28 at 20 06 57
Screenshot 2024-09-28 at 20 07 34

single para and double para doesn't make a difference, always get inline text, not even a line break

Screenshot 2024-09-28 at 20 07 24

### PR DESCRIPTION
**Pull request by hardingjam**
## Solution

Closes #30 
Add tailwind class to include line breaks

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: app/_components/FrameImage.tsx [hunk 1]
```diff
@@ -12,7 +13,7 @@ export const FrameImage = ({ currentState }: { currentState: FrameState }) => {
 			</div>
 			{currentState.deploymentOption ? (
 				<div
-					className="top-0 md:text-xl text-sm border rounded-full bg-gray-100 px-6 py-3 my-4 md:mb-0"
+					className="top-0 md:text-xl text-sm border rounded-full bg-gray-100 px-6 py-3 my-4 md:mb-0 "
```
Comment (thedavidmeister) [inline app/_components/FrameImage.tsx:16]: why add this whitespace?

**Review by thedavidmeister — CHANGES_REQUESTED** [review]
no screenshots on this PR

**Comment by hardingjam** [comment]
image

**Comment by hardingjam** [comment]
image

**Review by thedavidmeister — APPROVED** [review]

**Pull request merged**